### ISSUE TITLE
Get TFS WI with specified fields

### ISSUE THREAD

**Issue by allburov** [devopshq/tfs#14]
Sometimes we need get TFS WI with specified fields, because we sync large amounts of data.

Like in [jira](https://github.com/pycontribs/jira/tree/develop/jira):
```
query = jira.search_issues(jql_str="attachments is not EMPTY", json_result=True, fields="key, attachment")
```

**Comment by nobodyguy** [comment]
Have you tried WIQL?
https://github.com/devopshq/tfs#run-wiql

**Comment by allburov** [comment]
Yep, but ```### NOTE: Fields in SELECT really ignored, wiql return Work Items with all fields``` :(

**Comment by SAnCherepan** [comment]
Seems to work within the [get_workitems] method (https://github.com/devopshq/tfs/blob/master/tfs/connection.py#L74).

ToDo: add fields support for WIQL.
